uint16 command
---
bool isSucceeded